### ISSUE TITLE
ImageSimilarity error

### ISSUE THREAD

**Issue by rizr** [peterbraden/node-opencv#383]
when I am compare some image, I am receive error

OpenCV Error: Assertion failed (type == src2.type() && src1.cols == src2.cols && (type == CV_32F || type == CV_8U)) in batchDistance, file /build/buildd/opencv-2.4.8+dfsg1/modules/core/src/stat.cpp, line 2473
terminate called after throwing an instance of 'cv::Exception'
  what():  /build/buildd/opencv-2.4.8+dfsg1/modules/core/src/stat.cpp:2473: error: (-215) type == src2.type() && src1.cols == src2.cols && (type == CV_32F || type == CV_8U) in function batchDistance

**Comment by TanaseButcaru** [comment]
Did my pull request #381 solved your problem?

**Comment by rizr** [comment]
Yes, I am forked you PR, and hardcode in package.json, instead a full crash I am receive a NULL is better

**Comment by TanaseButcaru** [comment]
Great!
I couldn't test the case with the error, but now it's confirmed :+1:
